src/test/perl/NCM/Component/brokennew.pm

```diff
@@ -0,0 +1,12 @@
+package NCM::Component::brokennew;
+
+use strict;
+use warnings;
+use LC::Exception;
+
+# Needs to hav a new method, ideally via NCM::Component inheritance
```

Comment:
The code in `_load` doesn't check anything, i'll test whatever the code does :smile: